src/containers/CircleCI/List.js

```diff
@@ -0,0 +1,100 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Redirect } from 'react-router';
+import { Link } from 'react-router-dom';
+
+import moment from 'moment'
+import uuidv1 from 'uuid'
+
+import {
+  getBuildList,
+  selectBuild,
+} from '../../reducers/CircleCIReducer/actions';
+
+import './css/List.css'
+
+class List extends Component {
+
+  state = { 
+    redirect: false, 
+  }
+
+  componentDidMount() {
+    const apiKey = sessionStorage.getItem('CIRCLE_CI_API_KEY');
```

Comment:
It's a good idea to connect the api key in session storage to redux store, so you don't always access session storage